app/src/main/java/com/sunasterisk/fooddaily/ui/fragment/home/HomePresenter.kt

```diff
@@ -6,7 +6,7 @@ class HomePresenter (private val view: HomeContract.View): HomeContract.Presente
     override fun createDailyMenu(otherFoods: List<FoodDetail>) {
         view.showDailyMenu(
             mutableListOf<FoodDetail>().apply {
-                for (food in otherFoods) if (otherFoods.indexOf(food) % 3 == 0) add(food)
+                for (i in 5 until otherFoods.size) if (i % 3 == 0) add(otherFoods[i])
```

Comment:
5 là gì vậy?

Reply:
đoạn này em muốn là sẽ bắt đầu duyệt từ phần tử thứ 5 trở đi, check index để lấy ra một list con của list ban đầu để hiển thị trong daily menu. Em nhớ ở trong pull sau em có thay đổi lại thuật toán đoạn này một chút. Em tạo một mảng chứa index, sau đó em trộn mảng này lên, rồi lấy từ 5 phần tử đầu làm daily menu, nhưng em không nhớ mình để nó trong pull nào :3

Reply:
Nếu vậy thì để 5 ra thành 1 constant cho dễ hiểu đi em. Sau này còn có thể chỉnh sửa nữa